web/json_object_test.go

```diff
@@ -454,6 +454,10 @@ func TestConvertJSONObject_Flattening(t *testing.T) {
 						ExternalId: "a__a__a",
 						Type:       framework.AttributeTypeString,
 					},
+					{
+						ExternalId: "a__a__b",
+						Type:       framework.AttributeTypeString,
+					},
```

Comment:
I've added this attribute to show that we should be able to sync multiple fields in a complex valued attribute:
```json
"a": {
    "a": {
        "a": "a__a__a value",
		"b": "a__a__b value"
	},
}
```
We can sync both `a__a__a` and `a__a__b`.